<launch>

  <!-- Set namespace -->
  <arg name="ns_prefix" default="cam1" />
  
  
  <group ns="$(arg ns_prefix)">
      <param name="obstacle_range_limit" type="double" value="1.5" />
      <param name="width_regions" type="int" value="12" />
      <param name="height_regions" type="int" value="9" />
      <param name="max_distance_for_visualization" type="int" value="16"/>
      <param name="depth_topic" type="string" value="depth_image"/>
      <param name="camera_info_topic" type="string" value="camera_info"/>
      <param name="detect_topic" type="string" value="detect"/>
      <param name="obstacle_state_topic" type="string" value="obstacle_state"/>
      <param name="cam_id" type="string" value="cam1"/>
      <param name="verbose" type="bool" value="false" />
      <node pkg="depth_obstacle_detect_ros" type="depth_obstacle_detect_ros_node" name="ros_depth_obstacle_detect" output="screen">
      </node>
  </group>

</launch>
